js/ai/src/formats/types.ts

```diff
@@ -23,7 +23,9 @@ export type OutputContentTypes = 'application/json' | 'text/plain';
 
 export interface Formatter<O = unknown, CO = unknown> {
   name: string;
-  config: ModelRequest['output'];
+  config: ModelRequest['output'] & {
+    defaultInstruction?: false;
```

Comment:
```suggestion
    defaultInstructions?: false;
```

plural to match the `instructions` option